http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -537,21 +537,17 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
   ): F[Unit] =
     for {
       ctx <- ev.ask
+      _ <- logger.info(s"Beginning disk deletion for ${diskId} | trace id: ${ctx.traceId}")
       diskOpt <- persistentDiskQuery.getById(diskId).transaction
       disk <- diskOpt.fold(
         F.raiseError[PersistentDisk](PubsubHandleMessageError.DiskNotFound(diskId))
       )(F.pure)
-      _ <- if (disk.status != DiskStatus.Deleting)
```

Comment:
the original intention here was to verify the state did not change between front leo queuing the message and back leo receiving it. Its a bit duplicative though, because front leo also does this verification. We now also use this function for clean-up, so removing the status check.

Reply:
I'm trying to understand your comment. If the original intention was to verify the state did not change between front leo queuing the message and back leo receiving it then wouldn't this still be the right place to do it as front leo cannot do this verification by nature.

Reply:
Front Leo verifies that the disk can be deleted when it is requested. I don't know if there is a scenario in which the state can change between front leo messaging and back leo recieving it. Perhaps if an update request is made in the interim, but if a delete request is made and then an update request, I'm thinking Leo should still delete the disk